core/src/main/java/org/apache/calcite/adapter/jdbc/JdbcToEnumerableConverterRule.java

```diff
@@ -35,7 +35,7 @@ public JdbcToEnumerableConverterRule(JdbcConvention out,
       RelBuilderFactory relBuilderFactory) {
     super(RelNode.class, (Predicate<RelNode>) r -> true, out,
         EnumerableConvention.INSTANCE, relBuilderFactory,
-        "JdbcToEnumerableConverterRule:" + out);
+        "JdbcToEnumerableConverterRule");
```

Comment:
@wenhuitang Can you please help to make sure that all the `ConverterRule` sub-class all have the right description prefix ? Really thanks.

Reply:
Yes, of course, I will check all the sub-class of ConverRule.